file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,588 @@
+  "! Assigned content checks
+  TYPES ty_content_checks       TYPE SORTED TABLE OF ty_content_check WITH UNIQUE KEY check_id check_number.
+  "! <p class="shorttext">Filter Operation</p>
+  "! Operand to build the where condition
+  "! $values {@link zif_aff_edcc_v1.data:co_filter_operation}
+  "! $default {@link zif_aff_edcc_v1.data:co_filter_operation.and}
+  TYPES ty_filter_operation     TYPE c LENGTH 3.
+  "! <p class="shorttext">Comparison Operator</p>
+  "! Comparison operator
+  "! $values {@link zif_aff_edcc_v1.data:co_compare_operation}
+  "! $default {@link zif_aff_edcc_v1.data:co_compare_operation.equal}
+  TYPES ty_comparison_operator  TYPE c LENGTH 3.
+  "! <p class="shorttext">Filter Field Value</p>
+  "! Value used to build the where condition
+  TYPES ty_filter_value         TYPE c LENGTH 30.
+  "! <p class="shorttext">Expression Indicator</p>
+  "! Expression indicator
+  "! $values {@link zif_aff_edcc_v1.data:co_expression}
+  "! $default {@link zif_aff_edcc_v1.data:co_expression.begin_of}
+  TYPES ty_expression_indicator TYPE c LENGTH 3.
+  TYPES:
+    "! <p class="shorttext">Selection Filters on Check</p>
+    "! Selection filters on check
+    BEGIN OF ty_check_filter,
+      "! <p class="shorttext">Sequence Number</p>
+      "! Sequence number
+      sequence_number      TYPE ty_sequence_number,
+      "! <p class="shorttext">Filter Operation</p>
+      "! Operand to build the where condition
+      filter_operation     TYPE ty_filter_operation,
+      "! <p class="shorttext">Filter Field</p>
+      "! Field on which where condition would be applied
+      filter_field         TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Comparison Operator</p>
+      "! Comparison operator
+      comparison_operator  TYPE ty_comparison_operator,
+      "! <p class="shorttext">Filter Field Value</p>
+      "! Value used to build the where condition
+      filter_value         TYPE ty_filter_value,
+      "! <p class="shorttext">Expression Indicator</p>
+      "! Expression indicator
+      expression_indicator TYPE ty_expression_indicator,
+    END OF ty_check_filter.
+  "! <p class="shorttext">Selection Filters on Check</p>
+  "! Selection filters on check
+  TYPES ty_check_filters         TYPE SORTED TABLE OF ty_check_filter WITH UNIQUE KEY sequence_number.
+  "! <p class="shorttext">Position Number</p>
+  "! Position number
+  TYPES ty_check_position_number TYPE c LENGTH 3.
+  "! <p class="shorttext">Check Execution Sequence</p>
+  "! Check execution sequence
+  TYPES ty_execution_sequence    TYPE c LENGTH 3.
+  "! <p class="shorttext">Check Execution Control</p>
+  "! Check execution control
+  "! $values {@link zif_aff_edcc_v1.data:co_execution_control}
+  "! $default {@link zif_aff_edcc_v1.data:co_execution_control.proceed}
+  TYPES ty_execution_control     TYPE c LENGTH 2.
+  TYPES:
+    "! <p class="shorttext">Check Attribute</p>
+    "! Check attribute
+    BEGIN OF ty_check_attribute,
```

Comment:
There seems to be check attributes for the checks defined before. Is there a reason why you put the check attributes as a separate structure and not directly to the checks? For example as a sub structure?